index.html

```diff
@@ -0,0 +1,63 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta name="author" content="Sheyla Pozo" />
+    <title>Portfolio</title>
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.14.0/css/all.min.css"
+    />
+    <link rel="preconnect" href="https://fonts.gstatic.com" />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,100;0,300;0,700;1,200&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="./assets/style.css" />
+  </head>
+  <body>
+    <main>
+      <header>
+        <a href="#">My logo</a>
+        <nav>
+          <i class="fas fa-bars"></i>
+        </nav>
+      </header>
+      <section class="section-top">
+        <h1 class="section-title">I’m Sheyla Glad to see you!</h1>
+        <p>
+          I’m a software developer! I can help you build a product, feature or
+          website Look through some of my work and experience! If you like what
+          you see and have a project you need coded, don’t hestiate to contact
+          me.
+        </p>
+        <p><a href="#" class="main-text">LET’S CONNECT</a></p>
```

Comment:
- [ ] **OPTIONAL**
Good job making this text a link 👍 But, don't you think it'd be better semantically if we removed that `<p>` tag element since it's for paragraphs only? Please tell me what you think!

Reply:
sure gotta check up

Reply:
ok fixed it